src/models/page/page.ts

```diff
@@ -0,0 +1,67 @@
+import { market } from "../market";
+import { seoMetadata } from "../seoMetadata";
+import { ExpandedContentModel } from "../types/ExpandedContentModel";
+import { createField } from "../utils/createField";
+import { pageContentCustomerStory } from "./pageContentCustomerStory";
+import { pageContentExpert } from "./pageContentExpert";
+import { pageContentModular } from "./pageContentModular";
+import { pageContentNewsAndInsights } from "./pageContentNewsAndInsights";
+import { pageContentServiceDetails } from "./pageContentServiceDetails";
+import { pageContentSoftwareDetails } from "./pageContentSoftwareDetails";
+
+export const page = {
+  sys: { id: "page" },
+  name: "Page",
+  description:
+    "Represents the core data for each page on the site. Page URL is determined based upon market, the type of content provided, and then path fields.",
+  fields: [
+    createField("contentfulLabel"),
+    createField("shortText", {
+      id: "path",
+      name: "Path",
+      required: true,
+      editorInterface: {
```

Comment:
Since this is a key value for the entire project, shouldn't we add some validation to it?

```suggestion
      validations: [{ regexp: { pattern: "^[a-z0-9-]+(?:/[a-z0-9-]+)*$" } }],
      editorInterface: {
```

The pattern will allow paths like:
`about`
`software/products`
`services/consulting/strategy`

Reply:
Ah yes, I meant to figure this out before I readied this for review, how would you feel about...

`validations: [{ regexp: { pattern: "^[a-z0-9-]+([\/][a-z0-9-]+)*$" } }],`

This ensures that the string doesn't start with a `/`, cannot end with a `/`, and can't have multiple `/` in a row.